### ISSUE TITLE
no-raw-text: check `aria-label` attribute

### ISSUE THREAD

**Issue by Moeriki** [intlify/eslint-plugin-vue-i18n#277]
**What rule do you want to change?**

no-raw-text

**Does this change cause the rule to produce more or fewer warnings?**

More

**How will the change be implemented?**

Possibly with an option. But it could be enabled by default.

**Please provide some example code that this change will affect:**

```vue
<button aria-label="Close">
  <some-close-icon />
</button>
```

**What does the rule currently do for this code?**

Nothing.

**What will the rule do after it's changed?**

Raw text not allowed in `aria-label`.

**Comment by ota-meshi** [comment]
Thank you for this issue.

You can already check by using the `attributes` option.
https://eslint-plugin-vue-i18n.intlify.dev/rules/no-raw-text.html#options
